source/adios2/toolkit/sst/util/CMakeLists.txt

```diff
@@ -3,25 +3,27 @@
 # accompanying file Copyright.txt for details.
 #------------------------------------------------------------------------------#
 
-if(ADIOS2_HAVE_MPI)
-  set(maybe_adios2_core_mpi adios2_core_mpi)
-else()
-  set(maybe_adios2_core_mpi)
-endif()
+if (NOT ADIOS2_PIP_INSTALL)
```

Comment:
Should we do this conditionals in the parents CMake?

Reply:
What do you mean?  Just don't include the `util` subdirectory at all?

Reply:
yes

Reply:
Done